wordle-web/src/views/AboutView.vue

```diff
@@ -1,25 +1,34 @@
 <template>
   <v-sheet color="blue" height="170px" class="pt-10 my-5">
```

Comment:
If you want to, you can tie the color for this to your themes so the about page changes with the color theme, but it isn't required.